ha/core/system_health/system_health.py

```diff
@@ -105,7 +121,10 @@ def _update(self, component: str, comp_type: str, comp_id: str, healthvalue: str
                                 comp_type=comp_type, comp_id=comp_id)
         self.healthmanager.set_key(key, healthvalue)
 
-        # TODO: Check and if not present already, store the node map.
+        key = self._prepare_key(const.COMPONENTS.NODE_MAP.value, node_id=self.node_id)
```

Comment:
This is not the right place to update the node_map. It should be done just once in config phase of mini-provisioning.
@indrajitzagade

Reply:
Sure, lets move it there.

Reply:
node_map initialised in config phase of mini-provisioning.